src/IdentityManager/src/IdentityManager.API/IdentityManager.API.csproj

```diff
@@ -8,11 +8,10 @@
   </PropertyGroup>
```

Comment:
Remove extra src and identityManager. Final path should be `src/IdentityManager/WebApi/IdentityManager.WebApi.csproj`

In c# abbreviations use CamelCase as wekk. `API`->`Api`